pages/oauth/authorize.js

```diff
@@ -85,12 +85,22 @@ const OAuthAuthorizePage = () => {
               values={{ field: 'response_type', value: query['response_type'], expected: 'code' }}
             />
           </MessageBox>
+        ) : query['code_challenge'] && query['code_challenge_method'] !== 'S256' ? (
```

Comment:
This is to avoid the `plain` method being used, which is considered insecure. `S256` is a value from the RFC, which refers to SHA-256, as explained in the PR body.